src/config.h

```diff
@@ -131,7 +131,7 @@
 #endif
 #endif
 
-#if __GNUC__ >= 4
+#if __GNUC__ >= 4 && __GNUC_MINOR__ >= 5
```

Comment:
This condition will be false for 5.0. We don't need minor >= 5 in this case.

Reply:
Sorry I forgot that. It should be OK now. Thanks.